From: Paul Belanger <paul.belanger@polybeacon.com>
Subject: Fix rt(c)p set debug ip taking wrong argument
Origin: http://svn.asterisk.org/svn/asterisk/branches/1.6.2@273234
Bug: https://issues.asterisk.org/view.php?id=17469
Applied-Upstream: 1.6.2.10

This patch does not include two extra coding-style fixes included in the
original upstream commit.

diff --git a/main/rtp.c b/main/rtp.c
index 016d3b5..6ac4fe7 100644
--- a/main/rtp.c
+++ b/main/rtp.c
@@ -4601,7 +4601,7 @@ static char *rtp_do_debug_ip(struct ast_cli_args *a)
 	int port = 0;
 	char *p, *arg;
 
-	arg = a->argv[3];
+	arg = a->argv[4];
 	p = strstr(arg, ":");
 	if (p) {
 		*p = '\0';
@@ -4631,7 +4632,7 @@ static char *rtcp_do_debug_ip(struct ast_cli_args *a)
 	int port = 0;
 	char *p, *arg;
 
-	arg = a->argv[3];
+	arg = a->argv[4];
 	p = strstr(arg, ":");
 	if (p) {
 		*p = '\0';
-- 
1.7.1

